src/plugins/intel_gpu/src/kernel_selector/cl_kernels/sdpa_micro.cl

```diff
@@ -389,7 +403,15 @@ KERNEL(micro_sdpa)(OPTIONAL_SHAPE_INFO_ARG
 #endif
 
         /* Apply attention mask */
-#if WITH_ATTN_MASK
+#ifdef STATIC_SCALAR_ATTN_MASK_VALUE
+        const num_e = (ugemm_kq_c_type_block0 * ugemm_kq_c_type_block1) / SUBGROUP_SIZE;
+        #pragma unroll
+        for (int i = 0; i < sizeof(S_tile.x) / sizeof(S_tile.x[0]); i++) {
+                #pragma unroll
+                for (int j = 0; j < num_e; ++j)
+                    S_tile.x[i][j] = binary_add(S_tile.x[i][j], masked_scale);
+        }
```

Comment:
Can something like this be used instead?
```suggestion
    #define mask_add_op(x) ((x) + masked_scale)
    tile_elementwise(S_tile, mask_add_op);
```

Reply:
That's better, updated. Thanks!